apps/api/src/instrument-records/instrument-records.service.ts

```
@@ -276,6 +276,16 @@ export class InstrumentRecordsService {
     const createdSessionsArray: Session[] = [];
 
     try {
+      const subjectIdList = records.map((record) => {
+        const { subjectId: subjectId } = record;
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)